spring/src/main/java/org/axonframework/spring/authorization/CommandAuthorizationInterceptor.java

```diff
@@ -0,0 +1,48 @@
+package org.axonframework.spring.authorization;
+
+import org.axonframework.commandhandling.CommandMessage;
+import org.axonframework.messaging.InterceptorChain;
+import org.axonframework.messaging.MessageHandlerInterceptor;
+import org.axonframework.messaging.unitofwork.UnitOfWork;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.security.core.GrantedAuthority;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+
+/**
+ * Message interceptor that verifies authorization based on {@code @PreAuthorize} annotations on commands
+ *
+ * @author Roald Bankras
```

Comment:
Nit: please add `@since 4.10.0` here:

```suggestion
 * Message interceptor that verifies authorization based on {@code @PreAuthorize} annotations on commands.
 *
 * @author Roald Bankras
 * @since 4.10.0
```